packages/pros-core/src/task/mod.rs

```diff
@@ -299,7 +296,7 @@ map_errno! {
 ///
 /// This function will block the entire task, preventing concurrent
 /// execution of async code. When in an async context, it is recommended
-/// to use [`sleep`] instead.
+/// to use the `sleep` function in [`pros_async`](https://crates.io/crates/pros-async) instead.
```

Comment:
Maybe you should still link to the docs?
```suggestion
/// to use the [`sleep`](https://docs.rs/pros_async/sleep) function in [`pros_async`](https://crates.io/crates/pros-async) instead.
```
Might be worth checking whether its https://docs.rs/pros_async/sleep or https://docs.rs/pros-async/sleep

Reply:
Yeah thats a good idea. I'll check right now.

Reply:
Either works. Putting an `_` just redirects to `-`.